images/macos/toolsets/toolset-10.15.json

```diff
@@ -282,6 +282,7 @@
             "url" : "https://raw.githubusercontent.com/actions/go-versions/main/versions-manifest.json",
             "arch": "x64",
             "platform" : "darwin",
+            "variableTemplate" : "GOROOT_{0}_{1}_X64",
```

Comment:
We use '_' instead of camel case in toolset files. Rename "variableTemplate" -> "variable_template"

Reply:
Fixed it